products.js

```diff
@@ -30,20 +30,14 @@ const VIEWPORT_HEIGHT = 960;
  */
 function formatPrice(priceString){
   priceString = priceString.replace('$', '').replace(/([\s]|[^0-9$.-])/g, '');
-  if (priceString.includes('$')){
-    priceString = priceString.substr(priceString.indexOf('$') + 1);
-  }
-  return parseFloat(priceString);
+  return parseFloat(priceString.substr(priceString.indexOf('$') + 1));
```

Comment:
How would I account for the above case, as well a case where it gives me something like 199.99$ (the price is correct, but there's a dollar sign at the end) with just one pass through the string?